Tasks/Xcode/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "devDependencies": {
-    "vsts-task-lib": "0.8.5"
+    "vsts-task-lib": "0.9.14"
```

Comment:
This has to be updated to new published version of vsts-task-lib with the pipe command changes